### ISSUE TITLE
reseting dropdown list state before opening

### ISSUE THREAD

**Issue by ehudkaldor** [Semantic-Org/Semantic-UI#794]
i am using a dropdown list to select a country in a user registration form:

```
                <div class="ui selection dropdown" id="address_countryName">
                  <input type="hidden" name="countryName"></input>
                  <div class="default text">Country</div>
                  <i class="dropdown icon"></i>
                  <div class="menu ui transition hidden" id="countryList">
                    {#Countries}
                      <div class="item" data-value="{name}">{name}</div>
                    {/Countries}
                  </div>
                </div>
```

(the template is filled using dust.js)

my problem is the state of the list as it opens. it seems to persist across usages. so, if i select USA, next time i will open the list (using a new form) it will be at USA (and not at Afganistan, which is the first country). even worse, if i open an existing user and set the value with javascript: dropdown("set selected", value), the list seems to open at  the same place. so, if i start afresh and use 'set selected' to set the value to USA, the list will still open at Afganistan, and not at USA, where i would expect it. is there a way to do that?
to summarize, this is a two-fold question:
- how to position the list to the value already selected, and
- how to reset the list's position to top (for example, if no value if selected)
